model/src/payout_curve/quanto.rs

```diff
@@ -369,8 +407,12 @@ pub fn bankruptcy_price_short(initial_price: u64, leverage: Leverage) -> u64 {
 ///
 /// This is an absolute value. How to apply it in order to calculate the bankruptcy price will
 /// depend on the party's position.
-fn bankruptcy_price_shift(initial_price: u64, leverage: Leverage) -> u64 {
-    initial_price / leverage.0 as u64
+/// Note: the resulting bankruptcy price is rounded to 0 decimal points.
+fn bankruptcy_price_shift(initial_price: u64, leverage: Decimal) -> u64 {
+    (Decimal::from(initial_price) / leverage)
+        .round_dp_with_strategy(0, RoundingStrategy::ToZero)
+        .to_u64()
+        .expect("Bankruptcy price to fit into u64")
```

Comment:
🙃 I think it's clearer if we convert everything to `Decimal` first, then have the formula in a separate line, and finally convert the result into the return type. That's what I've tried to do in this file.